locales/en/faq/faq5.md

```diff
@@ -1 +1 @@
-You can connect to the address [https://reservazionicie.interno.gov.it](prenotazionicie.interno.gov.it) and type the name of your municipality, to check whether it uses the CIE Agenda online dating system. If yes, you can proceed to book the appointment directly from the site, without any login. Alternatively, you can always ask for an appointment online on the website of your municipality, or by going to one of the municipal offices.
\ No newline at end of file
+You can connect to the address [https://reservazionicie.interno.gov.it](https://prenotazionicie.interno.gov.it) and type the name of your municipality, to check whether it uses the CIE Agenda online dating system. If yes, you can proceed to book the appointment directly from the site, without any login. Alternatively, you can always ask for an appointment online on the website of your municipality, or by going to one of the municipal offices.
```

Comment:
```suggestion
You can connect to the address [https://reservazionicie.interno.gov.it](https://www.prenotazionicie.interno.gov.it) and type the name of your municipality, to check whether it uses the CIE Agenda online dating system. If yes, you can proceed to book the appointment directly from the site, without any login. Alternatively, you can always ask for an appointment online on the website of your municipality, or by going to one of the municipal offices.
```

Reply:
Fixed in [efebc9a](https://github.com/pagopa/io-app/pull/1721/commits/efebc9ace7434ea0b5e985a1871bb165dde5062a) 
Also changed the wrongly typed url description `[https://reservazionicie.interno.gov.it]`